lib/network_utils.pm

```diff
@@ -52,7 +52,7 @@ Returns if can ping worker host gateway
 sub can_upload_logs {
     my ($gw) = @_;
     $gw ||= $default_gw;
-    return script_run('ping -c 1 ' . $default_gw);
+    return (script_run('ping -c 1 ' . $default_gw) == 0);
```

Comment:
so you reverse usage, please change this place too https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/wickedbase.pm#L75